scripts/ci_build_and_test_in_container.sh

```diff
@@ -216,7 +216,7 @@ else
   if [[ ! -z "${DATA_BASENAME_WE}" ]]; then
     # Here we pack the installation.
     # The `--transform` parameter provides consistency between the tarball name and the unpacked folder.
-    or_die tar czf ${DATA_EXCHANGE_DIR}/${DATA_BASENAME_WE}.tar.gz --directory=${GEOSX_TPL_DIR}/.. --transform "s/^./${DATA_BASENAME_WE}/" .
```

Comment:
So here was the string joined failure culprit !! @matteofrigo5 @TotoGaz good catch :+1: